tests/regression/tests/REQUEST-933-APPLICATION-ATTACK-PHP/933160.yaml

```diff
@@ -6,23 +6,26 @@ meta:
   name: 933160.yaml
 tests:
   - test_title: 933160-1
-    desc: function call regexp
+    desc: |
+      PHP function call in query.
+      payload: chr(123)
     stages:
       - stage:
           input:
-            data: Shell%5fexec=bla
             dest_addr: 127.0.0.1
             headers:
               Host: localhost
               User-Agent: "OWASP CRS test agent"
               Accept: text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5
-            method: POST
+            method: GET
             port: 80
             uri: /?foo=chr%28123%29
```

Comment:
For all URI's, please use the method name as first path segment, e.g. `get`, to prevent httpbin from returning 404.

Reply:
uh, this is new to me! Done. Does it worth to mention it on the contribution guideline? https://coreruleset.org/docs/development/contribution_guidelines/#writing-tests

Reply:
Yes, that would make sense.